speakers/kristian-glass.md

```diff
@@ -0,0 +1,3 @@
+name: Kristian Glass
+---
+Etc
```

Comment:
@doismellburning Good point... but the only thing I can find in the talks spreadsheet says "Incoming".